packages/@lwc/integration-karma/test/api/CustomElementConstructor-getter/index.spec.js

```diff
@@ -10,6 +10,8 @@ import ReflectCamel from 'x/reflectCamel';
 import WithChildElmsHasSlot from 'x/withChildElmsHasSlot';
 import WithChildElmsHasSlotLight from 'x/withChildElmsHasSlotLight';
 
+const vFragBookend = process.env.API_VERSION > 59 ? '<!---->' : '';
```

Comment:
Nit: I'm wondering if we should have some sort of shared utility to manage the differences in behavior caused by API versioning features. That way we don't need to have this line pasted in every single test file.
```javascript
// Utility automatically checks process.env.API_VERSION
import { vFragBookend } from 'versionedFeaturesUtil';

// OR
import { getVFragBookend } from 'versionedFeaturesUtil';

const vFragBookend = getVFragBookend(process.env.API_VERSION);
```

Reply:
Or even forgo the need to have these tests know about VFrag implementation:
```javascript
import { vFragmentWrapped } from 'versionedFeaturesUtils';

expect(elm.innerHTML).toBe(vFragmentWrapped(''));
expect(elm.innerHTML).toBe(vFragmentWrapped('<span>Content</span>'));
expect(elm.innerHTML).toBe(vFragmentWrapped('<span>Content</span>', 3)); // 3 bookends on each end
```

Reply:
I'm on the fence about this. I kind of like the use of `vFragBookend`; it is more verbose, but it is very explicit. There's also the wrinkle of how Karma [pulls in files](https://github.com/salesforce/lwc/blob/5495668173bd65b24f5d86ca1ef96acaf4b75520/packages/@lwc/integration-karma/scripts/karma-configs/hydration/base.js#L33-L41), and how our existing test utils are [attached to the global object](https://github.com/salesforce/lwc/blob/8da8054b74a1144da207d976f74b360f474160a4/packages/@lwc/integration-karma/helpers/test-utils.js#L8). How would you feel about deferring this to a separate follow-up WI? I feel like this is worth discussing, but I don't want it to block getting a fix out for our partner teams.

Reply:
Totally fine with deferring the discussion and not blocking the fix. I marked it a nit for that exact reason.

Reply:
Created W-14450401, capturing this work.